### ISSUE TITLE
SVG icons for signals are hard to align

### ISSUE THREAD

**Issue by rooby** [FortAwesome/Font-Awesome#14295]
Most icons seem to be relatively centred within their element, however the signal icons are very bottom left oriented, which can make it pretty hard to align them nicely.

For example. signal-alt is quite well centred but signal-alt-3, signal-alt-2 and signal-alt-1 get gradually less centred.

If you aim to centre align these it can be quite difficult without specifying individual styles per icon, which is not ideal.

You can see this issue at a glance by going to https://fontawesome.com/icons?d=gallery&q=signal and hovering over the different icons and seeing where the icons sit relative to the hover background colour.

**Comment by InsanityMeetsHH** [comment]
In my opinion is there no issue. The most common way is to display the signal from left to right and not from center.
You can see this in smart phones for example. I think you are describing a special case.

**Comment by robmadole** [comment]
Yep, @rooby this is quite intentional. Does the explanation make sense?
